test/src/utils/FileSystem.test.js

```diff
@@ -21,4 +21,34 @@ contract('FileSystem', () => {
     FileSystem.exists(testDir.name).should.be.false
   })
 
+  it('can copy a file when the destination does not exist', function() {
+    var testDir = tmp.dirSync()
+    var sourceFilePath = `${testDir.name}/test`
+    var destinationFilePath = `${testDir.name}/testCopy`
+    FileSystem.write(sourceFilePath, 'Hello, World!')
+
+    FileSystem.copy(sourceFilePath, destinationFilePath)
+
+    var source = FileSystem.read(sourceFilePath).toString()
+    var destination = FileSystem.read(destinationFilePath).toString()
+    source.should.equal(destination)
+
+    FileSystem.removeTree(testDir.name)
+  })
+
+  it('can copy a file when the destination already exists', function () {
+    var testDir = tmp.dirSync()
+    var sourceFilePath = `${testDir.name}/test`
+    var destinationFilePath = `${testDir.name}/testCopy`
+    FileSystem.write(sourceFilePath, 'Hello, World!')
+    FileSystem.write(destinationFilePath, 'Foobar')
+
+    FileSystem.copy(sourceFilePath, destinationFilePath)
+
+    var source = FileSystem.read(sourceFilePath).toString()
+    var destination = FileSystem.read(destinationFilePath).toString()
+    source.should.equal(destination)
+
+    FileSystem.removeTree(testDir.name)
```

Comment:
can we use `const` here? feel free to update the rest of the variable declarations in this test file if you want

Reply:
Done :slightly_smiling_face: